### PR TITLE
Add NomisClient People and Moves libraries

### PR DESCRIPTION
**Pull request by cesidio**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -19,6 +19,9 @@ Layout/AlignParameters:
 Metrics/LineLength:
   Max: 120
 
+Metrics/MethodLength:
+  Max: 15
```
Comment (stevehook) [inline .rubocop.yml]: Depends how often we are hitting this limit but if it's infrequently we should probably stick with the default and opt-out on a per-method basis. I guess you hit it here where you are returning a multiline hash even though it's logically just one line of code.

File: app/lib/nomis_client/people.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        attributes_for(
+          NomisClient.get("/prisoners/#{prison_number}").parsed.first
+        )
+      end
+
+      def attributes_for(person)
+        {
+          nomis_prison_number: person['offenderNo'],
+          last_name: person['lastName'],
+          first_names: [person['firstName'], person['middleNames']].compact.join(' '),
```
Comment (stevehook) [inline app/lib/nomis_client/people.rb]: Makes sense to map the keys from camel to snake case but I don't think this kind of mapping stuff belongs in the client library. It's a separate concern.

File: .rubocop.yml (hunk 1 above)
Comment (cesidio) [reply .rubocop.yml]: correct. I can turn it off just for that instance.

File: app/lib/nomis_client/people.rb (hunk 2 above)
Comment (cesidio) [reply app/lib/nomis_client/people.rb]: I agree with that, will change it

**Pull request merged**